### PR TITLE
docs: update the operator on operatorhub.io

### PR DESCRIPTION
**Pull request by SteveMattar**
docs: add a new step for the release process on how to update the operator on operatorhub.io

Signed-off-by: Steve Mattar <smattar@redhat.com>

### CODE REVIEW

**Comment by SteveMattar** [comment]
Closes #141

File: src/content/contributing/release-process/_index.en.md [hunk 1]
```diff
@@ -212,11 +212,42 @@ If this is a final release, add a section for it on this website's [release note
 Alternatively you can edit the file and create a pull request directly on GitHub
 [here](https://github.com/submariner-io/submariner-website/edit/master/src/content/releases/_index.en.md)
 
-### Step 8: Verify the release
+### Step 8: Update submariner-operator on operatorhub.io
```
Comment (nyechiel) [inline src/content/contributing/release-process/_index.en.md:215]:
```suggestion
### Step 8: Update the Submariner operator on OperatorHub.io
```

File: src/content/contributing/release-process/_index.en.md [hunk 2]
```diff
@@ -212,11 +212,42 @@ If this is a final release, add a section for it on this website's [release note
...
+
+1) Clone the [submariner-operator](https://github.com/submariner-io/submariner-operator) project
+2) Make sure you have operator-sdk installed on your machine
+3) Generate a new csv file by running the command:
```
Comment (nyechiel) [inline src/content/contributing/release-process/_index.en.md:219]:
```suggestion
3) Generate a new CSV file by running the command:
```

File: src/content/contributing/release-process/_index.en.md [hunk 3]
```diff
@@ -212,11 +212,42 @@ If this is a final release, add a section for it on this website's [release note
...
+    --default-channel=true \
+    --operator-name=submariner \
+    --update-crds \
+    --make-manifests=false \
+    --interactive=false
+   ```
+
+   the output generated package should be located in `deploy/olm-catalog/submariner`
+4) Fork and clone [community-operators](https://github.com/operator-framework/community-operators) project.
+5) Update the kubernetes operator
```
Comment (nyechiel) [inline src/content/contributing/release-process/_index.en.md:234]:
```suggestion
5) Update the Kubernetes operator:
```

File: src/content/contributing/release-process/_index.en.md [hunk 4]
```diff
@@ -212,11 +212,42 @@ If this is a final release, add a section for it on this website's [release note
...
+    * copy the new generated package from step 3 into `upstream-community-operators/submariner`
+    * compare the new csv file with the previous one and update the missing fields. (e.g. spec.description)
```
Comment (nyechiel) [inline src/content/contributing/release-process/_index.en.md:236]:
```suggestion
    * compare the new CSV file with the previous one and update the missing fields (e.g spec.description)
```

File: src/content/contributing/release-process/_index.en.md [hunk 5]
```diff
@@ -212,11 +212,42 @@ If this is a final release, add a section for it on this website's [release note
...
+    * update the version in `upstream-community-operators/submariner/submariner.package.yaml`
+    * test the operator by running the command: `make operator.test OP_PATH=upstream-community-operators/submariner`
+    * preview the operator on [OperatorHub.io](https://operatorhub.io/preview)
+    * once everything is fine, review this
+    [checklist](https://github.com/operator-framework/community-operators/blob/master/docs/pull_request_template.md)
+    and create a new PR on [community-operators](https://github.com/operator-framework/community-operators)
+6) Update the openshift operator
```
Comment (nyechiel) [inline src/content/contributing/release-process/_index.en.md:243]:
```suggestion
6) Update the OpenShift operator:
```

File: src/content/contributing/release-process/_index.en.md [hunk 6]
```diff
@@ -212,11 +212,42 @@ If this is a final release, add a section for it on this website's [release note
 Alternatively you can edit the file and create a pull request directly on GitHub
 [here](https://github.com/submariner-io/submariner-website/edit/master/src/content/releases/_index.en.md)
 
-### Step 8: Verify the release
+### Step 8: Update submariner-operator on operatorhub.io
+
```
Comment (nyechiel) [inline src/content/contributing/release-process/_index.en.md:216]:
```suggestion
The Git repository called [community-operators](https://github.com/operator-framework/community-operators) is the source for community based operators. This repository is split into two sections:

* Operators for deployment to a vanilla Kubernetes environment (upstream-community-operators)
* Operators for deployment to OpenShift (community-operators)

operatorhub.io displays content that is targeted for vanilla Kubernetes.
```

**Review by nyechiel — COMMENTED** [review]
Thanks for looking into this, @SteveMattar. I added some suggestions, but overall LGTM

**Review by nyechiel — APPROVED** [review]

File: src/content/contributing/release-process/_index.en.md [hunk 7]
```diff
@@ -212,11 +212,50 @@ If this is a final release, add a section for it on this website's [release note
 Alternatively you can edit the file and create a pull request directly on GitHub
 [here](https://github.com/submariner-io/submariner-website/edit/master/src/content/releases/_index.en.md)
 
-### Step 8: Verify the release
+### Step 8: Update the Submariner operator on OperatorHub.io
+
+The Git repository called [community-operators](https://github.com/operator-framework/community-operators)
+is the source for community based operators. This repository is split into two sections:
+
+* Operators for deployment to a vanilla Kubernetes environment (upstream-community-operators)
```
Comment (tpantelis) [inline src/content/contributing/release-process/_index.en.md:220]:
```suggestion
* Operators for deployment to a vanilla Kubernetes environment (upstream-community-operators). These are shared with the Kubernetes community via [OperatorHub.io\(https://operatorhub.io/).
```

File: src/content/contributing/release-process/_index.en.md [hunk 8]
```diff
@@ -212,11 +212,50 @@ If this is a final release, add a section for it on this website's [release note
 Alternatively you can edit the file and create a pull request directly on GitHub
 [here](https://github.com/submariner-io/submariner-website/edit/master/src/content/releases/_index.en.md)
 
-### Step 8: Verify the release
+### Step 8: Update the Submariner operator on OperatorHub.io
+
+The Git repository called [community-operators](https://github.com/operator-framework/community-operators)
+is the source for community based operators. This repository is split into two sections:
```
Comment (tpantelis) [inline src/content/contributing/release-process/_index.en.md:218]:
```suggestion
The [community-operators](https://github.com/operator-framework/community-operators) Git repository
is the source for sharing Kubernetes Operators with the broader community. This repository is split into two sections:
```

File: src/content/contributing/release-process/_index.en.md [hunk 9]
```diff
@@ -212,11 +212,50 @@ If this is a final release, add a section for it on this website's [release note
...
+
+* Operators for deployment to a vanilla Kubernetes environment (upstream-community-operators)
+* Operators for deployment to OpenShift (community-operators)
+
+OperatorHub.io displays content that is targeted for vanilla Kubernetes.
```
Comment (tpantelis) [inline src/content/contributing/release-process/_index.en.md:223]:
```suggestion
To publish the Submariner operator to the community, perform the following stes:
```

File: src/content/contributing/release-process/_index.en.md [hunk 10]
```diff
@@ -212,11 +212,50 @@ If this is a final release, add a section for it on this website's [release note
...
+
+1) Clone the [submariner-operator](https://github.com/submariner-io/submariner-operator) project
+2) Make sure you have operator-sdk installed on your machine
```
Comment (tpantelis) [inline src/content/contributing/release-process/_index.en.md:226]:
```suggestion
2) Make sure you have the operator-sdk installed on your machine
```

Can you provide instructions on how to install the operator-sdk and what version?

File: src/content/contributing/release-process/_index.en.md [hunk 11]
```diff
@@ -212,11 +212,50 @@ If this is a final release, add a section for it on this website's [release note
...
+    --csv-version=${OPERATOR_VERSION} \
+    --csv-channel=alpha \
+    --default-channel=true \
+    --operator-name=submariner \
+    --update-crds \
+    --make-manifests=false \
+    --interactive=false
+   ```
+
+   the output generated package should be located in `deploy/olm-catalog/submariner`
```
Comment (tpantelis) [inline src/content/contributing/release-process/_index.en.md:240]:
```suggestion
   the generated package output should be located in `deploy/olm-catalog/submariner`
```

File: src/content/contributing/release-process/_index.en.md [hunk 12]
```diff
@@ -212,11 +212,50 @@ If this is a final release, add a section for it on this website's [release note
...
+4) Fork and clone [community-operators](https://github.com/operator-framework/community-operators) project.
+5) Update the Kubernetes operator:
+    * copy the new generated package from step 3 into `upstream-community-operators/submariner`
```
Comment (tpantelis) [inline src/content/contributing/release-process/_index.en.md:243]:
```suggestion
    * copy the generated package from step 3 into `upstream-community-operators/submariner`
```

File: src/content/contributing/release-process/_index.en.md [hunk 13]
```diff
@@ -212,11 +212,50 @@ If this is a final release, add a section for it on this website's [release note
...
+    * compare the new CSV file with the previous one and update the missing fields (e.g spec.description)
+    * update the version in `upstream-community-operators/submariner/submariner.package.yaml`
+    * test the operator by running the command: `make operator.test OP_PATH=upstream-community-operators/submariner`
+    * preview the operator on [OperatorHub.io](https://operatorhub.io/preview)
+    * once everything is fine, review this
+    [checklist](https://github.com/operator-framework/community-operators/blob/master/docs/pull_request_template.md)
+    and create a new PR on [community-operators](https://github.com/operator-framework/community-operators)
+6) Update the OpenShift operator:
+   * run step 5 again but this time on `community-operators/submariner` directory.
```
Comment (tpantelis) [inline src/content/contributing/release-process/_index.en.md:252]:
```suggestion
   * re-run step 5 on the `community-operators/submariner` directory.
```

File: src/content/contributing/release-process/_index.en.md [hunk 14]
```diff
@@ -212,11 +212,50 @@ If this is a final release, add a section for it on this website's [release note
 Alternatively you can edit the file and create a pull request directly on GitHub
 [here](https://github.com/submariner-io/submariner-website/edit/master/src/content/releases/_index.en.md)
 
-### Step 8: Verify the release
+### Step 8: Update the Submariner operator on OperatorHub.io
```
Comment (tpantelis) [inline src/content/contributing/release-process/_index.en.md:215]: I think this should go after the "Verify the release" step.

**Review by tpantelis — CHANGES_REQUESTED** [review]

File: src/content/contributing/release-process/_index.en.md (hunk 10 above)
Comment (tpantelis) [reply src/content/contributing/release-process/_index.en.md]: 👆

Comment (SteveMattar) [reply src/content/contributing/release-process/_index.en.md]: Sorry I missed that comment

**Review by tpantelis — APPROVED** [review]

**Pull request merged**